### PR TITLE
Send authentication data via headers instead of query params

### PR DESCRIPTION
**Pull request by kjbenk**

### CODE REVIEW

File: lib/strategy.js [hunk 1]
```diff
@@ -81,6 +81,13 @@ util.inherits(Strategy, OAuth2Strategy);
  * @api protected
  */
 Strategy.prototype.userProfile = function (accessToken, done) {
+  /**
+   * Ensure the authorization is passed via the headers instead of
+   * query params since our Laravel Passport validation logic does
+   * not accept the `access_token` param for authentication.
+   */
+  this._oauth2.useAuthorizationHeaderforGET(true);
```
Comment (mgriego) [inline lib/strategy.js:89]: 🎉

**Review by mgriego — APPROVED** [review]

**Comment by mgriego** [comment]
I will say that, since we're already updating this client, we should probably go ahead and update the default URLs to point to the Atlas SSO endpoints......

**Comment by kjbenk** [comment]
Default URLs updated :)

File: lib/strategy.js [hunk 2]
```diff
@@ -44,14 +44,14 @@ Examples:
 **/
 function Strategy(options, verify) {
   options = options || {};
-  options.authorizationURL = options.authorizationURL || 'http://connect.sheknows.com/authorize';
-  options.tokenURL = options.tokenURL || 'http://connect.sheknows.com/token';
+  options.authorizationURL = options.authorizationURL || 'http://atlas-sso.revtech.internal/oauth/authorize';
+  options.tokenURL = options.tokenURL || 'http://atlas-id-manager-sso/oauth/token';
```
Comment (mgriego) [inline lib/strategy.js]: Let's default all of these to the public prod URLs: https://atlas-sso.pmc.com/...

**Review by mgriego — CHANGES_REQUESTED** [review]
Can we also go ahead and update the version (2.0.0) and description (Atlas SSO) in the `package.json`?

File: lib/strategy.js (hunk 2 above)
Comment (kjbenk) [reply lib/strategy.js]: Done

File: package.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "passport-sheknows",
-  "version": "1.0.1",
+  "version": "2.0.0",
   "license": "MIT",
-  "description": "Passport adapter for SheKnows Connect",
+  "description": "Atlas SSO",
```
Comment (mgriego) [inline package.json]:
```suggestion
  "description": "Passport adapter for Atlas SSO",
```

**Review by mgriego — CHANGES_REQUESTED** [review]

**Review by mgriego — APPROVED** [review]

**Pull request merged**